src/jit/codegenarm.cpp

```diff
@@ -1720,7 +1720,6 @@ void CodeGen::genEmitHelperCall(unsigned helper, int argSize, emitAttr retSize,
     }
 
     regTracker.rsTrashRegSet(RBM_CALLEE_TRASH);
-    regTracker.rsTrashRegsForGCInterruptability();
```

Comment:
This call from non-legacy code had no effect, `rsTrashRegsForGCInterruptability` only updates `RegTracker::rsRegValues` and that's not used in non-legacy code.